pulp_2to3_migration/app/plugin/rpm/pulp_2to3_models.py

```diff
@@ -723,9 +723,13 @@ def pre_migrate_content_detail(cls, content_batch):
             for distribution in pulp2_distribution_content_batch]
         cls.objects.bulk_create(pulp2distribution_to_save, ignore_conflicts=True)
 
-    def create_pulp3_content(self):
+    def get_treeinfo_serialized(self):
```

Comment:
This is just a refactor right, I'm not sure I see any functional changes in this section?

Reply:
The goal was to avoid instantiating a DistributionTree object if I need just a serialized metadata. So it is a refactor without logic changes in this place.

Reply:
+1